package.json

```diff
@@ -3,15 +3,14 @@
   "version": "1.0.0",
   "description": "A bot to help encourage Coding Buddies",
   "exports": "./index.js",
-  "type": "module",
   "node": ">=16.0.0",
+  "type": "module",
```

Comment:
lmao, i just went down a rabbithole, wondering how the hell you were going to use ESM without the `type: module` option in the package.json.

Then i came back and saw that its still there 🤦🏾‍♂️